tasks/publish_metrics_to_elasticsearch.rb

```diff
@@ -0,0 +1,6 @@
+task :publish_metrics_to_elasticsearch do
+  Volumetrics::UseCase::SyncS3ToElasticsearch.new(
```

Comment:
Should this be calling your new UseCase? That's the sync one from my bit...

Reply:
oh too much cut and paste. good catch